customer-service/build.gradle

```diff
@@ -15,12 +14,17 @@ java {
 
 dependencies {
 	implementation project(':dto')
-
 	implementation 'com.google.protobuf:protobuf-java-util:3.25.1'
-
+	implementation 'org.springframework.data:spring-data-commons:3.2.1'
+	compileOnly 'org.projectlombok:lombok:1.18.30'
```

Comment:
You have added lambok here

Reply:
https://spring.io/guides/gs/accessing-data-jpa/

Reply:
https://thorben-janssen.com/lombok-hibernate-how-to-avoid-common-pitfalls/